docs/en/authentic-sources.rst

```diff
@@ -6,7 +6,24 @@ Authentic Sources
 
 Authentic Sources are responsible for the authenticity of the User's attributes provided as Digital Credentials by Credential Issuers. During the Issuance Flow, Credential Issuers request from Authentic Sources the attributes required to provide the requested Credential. Authentic Sources MAY also provide a Credential Offer related to their Credential Issuers as defined in Section :ref:`Credential Offer Endpoint <Credential Offer Endpoint>`.
```

Comment:
```suggestion
Authentic Sources provides the User's attributes allowing the Credential Issuers to issue User's Digital Credentials. During the Issuance Flow, Credential Issuers request from Authentic Sources the attributes required to provide the requested Credential. Authentic Sources MAY also provide a Credential Offer related to their Credential Issuers as defined in Section :ref:`Credential Offer Endpoint <Credential Offer Endpoint>`.
```

the responsability of the authenticity of the data contained within the credentials are only up to the credential issuers and not to the authentic sources